<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="scene">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>1000</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_1">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>0</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>660</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseColorButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходное положение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Подсказка</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="panel_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>110</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>220</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_4">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>330</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_5">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>440</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>550</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_7">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>0</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_8">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>110</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_9">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>220</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_10">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>330</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_11">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>440</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="panel_12">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>550</y>
     <width>120</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
